esmvaltool/recipes/ref/recipe_model_benchmarking_boxplots_region_trend.yml

```diff
@@ -0,0 +1,177 @@
+# ESMValTool
+---
+documentation:
+  title: Benchmarking of a single model for a region through extract_shape.
+
+  description: |
+    Benchmarking: Box plots, linear trends.
+    The diagnostic called in this recipe uses the seaborn.boxplot function.
+    See seaborn manual for detailed information
+    (https://seaborn.pydata.org/generated/seaborn.boxplot.html).
+
+  authors:
+    - lauer_axel
+    - bock_lisa
+    - hassler_birgit
+    - lindenlaub_lukas
+    - schlund_manuel
+    - weigel_katja
+
+  maintainer:
+    - bock_lisa
+
+  references:
+    - lauer25gmd
+
+  projects:
+    - dlrmabak
+
+datasets:
+  # Dataset to be benchmarked
+  - {dataset: MIROC6, grid: gn, benchmark_dataset: true}
+
+VAR_SETTINGS: &var_settings
+  project: CMIP6
+  mip: Amon
+  exp: historical
+  ensemble: r1i1p1f1
+  start_year: 2001
```

Comment:
Could we also have an earlier start date here? The historical simulations have the data, and the obs should have it as well, no?

Reply:
Yes, but long time series I would include last (see also time series).